src/EnergyPlus/GeneratorDynamicsManager.cc

```diff
@@ -229,17 +223,8 @@ namespace GeneratorDynamicsManager {
         Real64 MaxPel;                          // working variable for max allowed by transient constraint
         Real64 MinPel;                          // working variabel for min allowed by transient constraint
         Real64 PelInput;                        // holds initial value of IN var
-        Real64 Pel;
         DataGenerators::OperatingMode newOpMode(DataGenerators::OperatingMode::Invalid);
-        Real64 SchedVal;
         Real64 ElectLoadForThermalRequest;
-        bool ConstrainedMaxP;           // true if request was altered because of max power limit
-        bool ConstrainedMinP;           // true if request was altered because of min power limit
-        bool ConstrainedIncreasingPdot; // true if request was altered because of power rate of change up
-        bool ConstrainedDecreasingPdot; // true if request was altered because of power rate of change down
-        bool ConstrainedByPlant;        // true if request was altered because of cooling water problem
-        bool PLRStartUp;                // true if subtimestep issue involving startup
-        bool PLRShutDown;
         auto &InletCWnode = state.dataGenerator->InletCWnode;
```

Comment:
These are written to, so they should not be shortcuts at all.

Reply:
OK, for one I don't understand this comment, we write to a reference all the time. And two, what's the fix here? Delete this reference and replace with state.dataGenerator->InletCWnode since it's only used twice? This should have been an int, not auto, but that's moot now since it looks like this should be removed.

Reply:
Yep, expand into `state.dataGenerator->InletCWnode`. Don’t write through local scalar reference shortcuts because those writes look like local variables.